exporters/prometheus/test/exporter_utils_test.cc

```diff
@@ -48,11 +48,13 @@ void assert_basic(prometheus_client::MetricFamily &metric,
     }
     break;
     case prometheus_client::MetricType::Summary:
-      // Summary type not supported
+      // Summary and Info type not supported
       ASSERT_TRUE(false);
       break;
     case prometheus::MetricType::Untyped:
       break;
+    default:
```

Comment:
v.1.1.0 adds a new metric type as MetricType::Info. Adding `default` to handle that (and keep maintainer mode CI happy).